.gitignore

```diff
@@ -12,3 +12,6 @@ _ReSharper.*
 .vs
 *.swp
 *.swo
+/packages/CommandLineParser.1.9.71

```

Comment:
Should we just not add packages folder here? Personally I prefer adding the dll's in too here. But upto @mikehadlow to decide